src/libretro/libretro-common/rthreads/psp_pthread.h

```diff
@@ -26,6 +26,7 @@
 
 #ifdef VITA
 #include <psp2/kernel/threadmgr.h>
+#include <sys/time.h>
```

Comment:
should be done to https://github.com/libretro/libretro-common/blob/master/rthreads/psp_pthread.h also?